app/src/main/java/swati4star/createpdf/activity/ImageEditor.java

```diff
@@ -153,7 +154,7 @@ private void changeAndShowImageCount(int count) {
 
         mCurrentImage = count % mDisplaySize;
         photoEditorView.getSource()
-                .setImageBitmap(BitmapFactory.decodeFile(mImagePaths.get(mCurrentImage)));
```

Comment:
I am curious why we can't use Picasso to show image in `photoEditorView`. Please give picasso a try...